src/module.ts

```diff
@@ -71,7 +71,7 @@ export default defineNuxtModule<ModuleOptions>({
       origin: usedOrigin
     })
 
-    const url = joinURL(options.origin, options.basePath)
+    const url = joinURL(options.origin ? options.origin : 'http://localhost:3000', options.basePath || '')
```

Comment:
why is this necessary @zoey-kaiser?

both `options.origin` and `options.basePath` are set to their default values above.